diff --git a/BBSR/DuctedFanSwirl.m b/BBSR/DuctedFanSwirl.m
index 1e8157d..a05be00 100755
--- a/BBSR/DuctedFanSwirl.m
+++ b/BBSR/DuctedFanSwirl.m
@@ -103,12 +103,12 @@ clear; clc; close all; eVTOL;
 % New IPM5 designs for AIAA paper
 % d.T = 50; d.u0 = 0; d.alt = 0; df = 0.43;
 % d.T = 10; d.u0 = 20; d.alt = 0; df = 0.35;
-d.T = 10; d.u0 = 10; d.alt = 0; df = 0.4;
+d.T = 50; d.u0 = 10; d.alt = 0; df = 0.4;
 igv1 = 0; igv2 = 0; sig = 1; rotation = 'N';
 ar2 = 2.2; r_cas = 0.143/2; ht = (44/1000)/(r_cas*2); nv = 0.5; k = 1;
-% phi = 0.60; des_name = 'IPM5_PH60D'; ar1 = 2.8;
-% phi = 0.75; des_name = 'IPM5_PH75D'; ar1 = 2.5;
-phi = 0.9; des_name = 'IPM5_PH90D'; ar1 = 2.5;
+ phi = 0.60; des_name = 'IPM5_PH60D'; ar1 = 2.8;
+ %phi = 0.75; des_name = 'IPM5_PH75D'; ar1 = 2.5;
+%phi = 0.9; des_name = 'IPM5_PH90D'; ar1 = 2.5;
 
 % New IPM5 designs for AIAA paper - with constant diffusion factor
 % % d.T = 50; d.u0 = 0; d.alt = 0; 
@@ -236,7 +236,7 @@ J = d.d.u0 / (n * D_tip);
 disp([J Ct])
 A = pi * (r_cas^2 - d.g.rhub^2); Um = 0.5 * pi * d.d.rpm * (d.g.rhub + r_cas) / 30;
 d.d.T / (d.d.atm.ro * A * Um^2)
-
+disp(Um)
 % Close even numbered figures
 % if mod(o,2) == 0; close(h); end;
 
@@ -246,6 +246,6 @@ if o ~= length(phi); clear d; clear CRF; end;
 end
 
 % Save designs
-if o == numel(phi)
-    save([dr.geom set_name '.mat'],'phi','sig','r_cas','df','ht','ar1','ar2','igv1','igv2');
-end
+% if o == numel(phi)
+%     save([dr.geom set_name '.mat'],'phi','sig','r_cas','df','ht','ar1','ar2','igv1','igv2');
+% end
diff --git a/BBSR/correct_pitch.m b/BBSR/correct_pitch.m
index 57f6106..56ccf55 100755
--- a/BBSR/correct_pitch.m
+++ b/BBSR/correct_pitch.m
@@ -39,6 +39,9 @@ else
     ks{1} = [1 1 17]; ks{2} = [53 53 17]; ks{3} = [1 1 17]; ks{4} = [1 1 17];
     bids_row{1} = 0:6; bids_row{2} = 7:13; bids_row{3} = 14:20; bids_row{4} = 21:27;
 
+
+
+    MAKE YOUR BIUDS NHERE
 end
 
 % Loop over both rows and correct pitch with extra points
diff --git a/BBSR/eVTOL.m b/BBSR/eVTOL.m
deleted file mode 100755
index bcf20c5..0000000
--- a/BBSR/eVTOL.m
+++ /dev/null
@@ -1,104 +0,0 @@
-%   eVTOL
-%                               
-%   Load eVTOL environment variables for CFD and experiment
-%
-%   Written by James Taylor & Sam Grimshaw May 2020
-
-
-%% Set directories and job variables
-
-% Root directory depending on host
-[~,host] = system('hostname'); host = strtrim(host);
-if strcmp(host,'rjm-125.eng.cam.ac.uk') == 1
-    dr.root = '/home/jvt24/eVTOL/';
-    job.user = 'jvt24'; 
-    dr.hpc = '/rds/project/amjd2/rds-amjd2-compressors/eVTOL/';
-elseif any(strcmp(host,{'DESKTOP-4OIBD51','WL-JT-004'})) == 1
-    dr.root = 'C:\Users\jvt24\Documents\eVTOL\';
-    job.user = 'jvt24';
-    dr.hpc = '/rds/project/amjd2/rds-amjd2-compressors/eVTOL/';
-elseif strcmp(host,'DESKTOP-DBFN3GS') == 1
-    dr.root = 'C:\Users\James\Documents\eVTOL\';
-    job.user = 'jvt24';
-    dr.hpc = '/rds/project/amjd2/rds-amjd2-compressors/eVTOL/';
-elseif strcmp(host,'WL-JT-002') ==1
-    dr.root = '/home/jc2071/eVTOL/';
-    job.user = 'jc2071';
-    dr.hpc = '/rds/hpc-work/eVTOL/';
-elseif strcmp(host,'WL-GP-005') ==1
-    dr.root = 'C:\Users\sdg33\OneDrive - University Of Cambridge\OtherWork\eVTOL\Blue Bear\dummy\';
-    job.user = 'sdg33';
-    dr.hpc = '/rds/hpc-work/eVTOL/';
-end
-
-
-% Sub directories saved in struct
-dr.ts = [dr.root 'TURBOSTREAM' filesep];
-dr.exp = [dr.root 'Exp' filesep];
-dr.geom = [dr.root 'Geometry' filesep];
-dr.pv = [dr.root 'PARAVIEW' filesep];
-dr.ag = [dr.root 'AUTOGRID' filesep];
-dr.fig = [dr.root 'Documents' filesep];
-dr.cad = [dr.root 'CAD' filesep];
-dr.nc = [dr.root 'NC' filesep];
-
-% Variables for Wilkes 3
-job.host = 'login-e-1.hpc.cam.ac.uk';
-job.directory = dr.ts;
-job.hpcdir = dr.hpc;
-job.version = '3.6.10';
-job.ppn = 4;
-job.arch = 'ampere';
-
-
-%% Fluid properties
-
-% Air properties for experiments
-exp_air
-
-% Gas constants
-cp = 1003.308; ga = 1.40047; mu = 1.8000e-05;
-cv = cp / ga; R = cp - cv;
-
-% Global variables for post processing CFD
-global Pref Tref
-Pref = 101325.8;
-Tref = 288.15;
-
-
-%% CFD mesh parameters
-
-% Boundary conditions
-I.pout = Pref; I.inlet = [dr.ts 'inlet_2percent.dat']; I.guess = [1.1 50 0 0 2e5 0.0002];
-
-% Iteration counts
-I.nstep = 100000; I.poisson_nstep = 10000; 
-
-% Preallocate cell arrays
-B = cell(2,1); F = cell(2,1); D = cell(2,1);
-
-% Define fillet parameters
-s_cl = [0 1]; rad = [1 1] * 8e-3;
-for n = 1:2
-    F{n}.hub.s1 = s_cl; F{n}.hub.s2 = s_cl; F{n}.hub.rad1 = rad; F{n}.hub.rad2 = rad; 
-    F{n}.cas.s1 = s_cl; F{n}.cas.s2 = s_cl; F{n}.cas.rad1 = rad; F{n}.cas.rad2 = rad; 
-    F{n}.flag = [2 2]; F{n}.tol = 1e-6;
-end
-
-% Define gap parameters
-D{1}.flag = [0 0]; D{2}.flag = [0 0];
-
-% Global mesh parameters for ducted fan
-M.dx_row = 20e-3;                                                   % Axial constant
-M.n_zi = [33 13]; M.n_zo = [13 33];                                 % Z-constant lines
-M.n_j = 81; M.n_jc = 30;                                            % Flow path and constant
-M.n_g = 25; M.n_f = 17; M.n_fb = 13; M.n_gb = 13;                   % Gap, fillet and butterfly
-M.n_bu = 65; M.n_bl = 65; M.n_b = 25; M.r_b = 1.15;                 % Blade blocks
-M.n_ai = 21; M.n_aiu = 17; M.n_aid = 17;                            % Azimuthal inlet block
-M.n_ao = 21; M.n_aou = 17; M.n_aod = 17;                            % Azimuthal outlet block
-M.n_si = 17; M.n_so = 17;                                           % Streamwise blocks
-M.e_j = 1.3e-5; M.e_b = 1e-5; M.e_f = 2e-3; M.e_s = 4e-5;           % First point spacing
-M.i_j = 2;                                                          % Spanwise interpolation
-M.skew = 12;                                                        % Skewness limit
-
-
